index.js

```diff
@@ -287,6 +287,7 @@ NYC.prototype._handleSourceMap = function (cacheDir, code, hash, filename) {
 
 NYC.prototype._handleJs = function (code, filename) {
   var relFile = path.relative(this.cwd, filename)
+  filename = path.resolve(this.cwd, relFile) // ensure the path has correct casing (see https://github.com/istanbuljs/nyc/issues/269)
```

Comment:
> What are your thoughts on the "re-resolve"?

tbh, they are kind limited to “this is not obviously wrong” and “the CI seems to be happy”, it’s not really a part of the codebase I have worked with a lot. :D